### ISSUE TITLE
Add security.txt conforms to RFC9116

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#7388]
```txt
Contact: mailto:support@radiasoft.net
Expires: 2025-03-14T00:00:00.000Z
```

Expires is just some time in the future. Make +1y

**Comment by moellep** [comment]
Current implementation has a leap-year bug.

**Comment by robnagler** [comment]
Fix 5e611a76a2f3701bdf2f7de9d27399ca4b06cceb
